atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -43,14 +45,62 @@ def __init__(self, tasker, workflow, registries):
 
         self.registries = deepcopy(registries)
 
+    def need_skopeo_push(self):
+        if len(self.workflow.exported_image_sequence) > 0:
+            last_image = self.workflow.exported_image_sequence[-1]
+            if last_image['type'] == IMAGE_TYPE_OCI or last_image['type'] == IMAGE_TYPE_OCI_TAR:
+                return True
+
+        return False
+
+    def push_with_skopeo(self, registry_image, insecure, docker_push_secret):
+        # If the last image has type OCI_TAR, then hunt back and find the
+        # the untarred version, since skopeo only supports OCI's as an
+        # untarred directory
+        index = 1
```

Comment:
I think `next(` version is fine as long as there is comment explaining what's happening there